index.html

```diff
@@ -55,5 +55,162 @@ <h2 class="header-subtitle">I’m a Software Developer</h2>
+          </p>
+        </div>
+        <ul class="pro-tag">
+          <li class="pro-h">html</li>
+          <li class="pro-h">bootstrap</li>
+          <li class="pro-h">Ruby</li>
+        </ul>
+           <button class="see-proj" type="button">See Project</button>
+      </div>
+      <div class="professional">
+        <div class="pro-art">
+         <h2 class="art-print">Profesional Art Printing Data</h2> 
+          <p class="daily-selection">A daily selection of privately personalized reads;
+            no accounts or sign-ups required. has been the industry's standard
+          </p>
+        </div>
+        <ul class="pro-tag">
+          <li class="pro-h">html</li>
+          <li class="pro-h">bootstrap</li>
+          <li class="pro-h">Ruby</li>
+        </ul>
+           <button class="see-proj" type="button">See Project</button>
+      </div>
+      <div class="professional">
+        <div class="pro-art">
+         <h2 class="art-print">Profesional Art Printing Data</h2> 
+          <p class="daily-selection">A daily selection of privately personalized reads;
+            no accounts or sign-ups required. has been the industry's standard
+          </p>
+        </div>
+        <ul class="pro-tag">
+          <li class="pro-h">html</li>
+          <li class="pro-h">bootstrap</li>
+          <li class="pro-h">Ruby</li>
+        </ul>
+           <button class="see-proj" type="button">See Project</button>
+      </div>
+      <div class="professional">
+        <div class="pro-art">
+         <h2 class="art-print">Profesional Art Printing Data</h2> 
+          <p class="daily-selection">A daily selection of privately personalized reads;
+            no accounts or sign-ups required. has been the industry's standard
+          </p>
+        </div>
+        <ul class="pro-tag">
+          <li class="pro-h">html</li>
+          <li class="pro-h">bootstrap</li>
+          <li class="pro-h">Ruby</li>
+        </ul>
+           <button class="see-proj" type="button">See Project</button>
+      </div>
+      <div class="professional">
+        <div class="pro-art">
+         <h2 class="art-print">Profesional Art Printing Data</h2> 
+          <p class="daily-selection">A daily selection of privately personalized reads;
+            no accounts or sign-ups required. has been the industry's standard
+          </p>
+        </div>
+        <ul class="pro-tag">
+          <li class="pro-h">html</li>
+          <li class="pro-h">bootstrap</li>
+          <li class="pro-h">Ruby</li>
+        </ul>
+           <button class="see-proj" type="button">See Project</button>
+      </div>   
+    </section>
+
+    <section class="about" id="about-menu">
+      <div class="about-me">
+        <div class="about-soft">
+          <h2 class="about-head">About me</h2>
+          <p class="hello">Hello I’m a software developer! I can help you build a product , feature or website Look through
```

Comment:
- [ ] please ensure the about me text follows the same font family and size of the Figma version.
- [ ] please center the `about me` heading
- [ ] The get my resume button should be bigger, and it uses the color  `#FF6B00`
yours
![Screenshot (309)_LI](https://user-images.githubusercontent.com/65298048/185485454-21fd751c-8c66-498b-9418-2ac0d51905a2.jpg)
figma
![Screenshot (310)_LI](https://user-images.githubusercontent.com/65298048/185485640-ff550270-e333-45ca-a51e-721c5e99eea3.jpg)